src/modules/cache-manager/cache-manager.service.ts

```diff
@@ -1,12 +1,14 @@
-import { Inject, Injectable, Logger } from '@nestjs/common';
+import { Inject, Injectable } from '@nestjs/common';
 import { CACHE_MANAGER } from '@nestjs/cache-manager';
 import { Cache } from 'cache-manager';
 
 @Injectable()
 export class CacheManagerService {
   private readonly USAGE_KEY_PREFIX = 'usage:';
   private readonly LIMIT_KEY_PREFIX = 'limit:';
+  private readonly JWT_KEY_PREFIX = 'jwt:';
   private readonly TTL_10_MINUTES = 10000 * 60;
+  private readonly TTL_7_DAYS = 604800;
```

Comment:
I assume this 7 days are due to the max possible JWT duration allowed on the login. I recommend binding that value to the jwt sign one by using the same number in the [expiresIn](https://www.npmjs.com/package/jsonwebtoken) value when signing a token, as it also accepts a number